### DISCUSSION TITLE
Use System Default Program on File

### DISCUSSION THREAD

**Discussion by HiramTheHero** [nvim-neo-tree/neo-tree.nvim#1097] [Q&A]
Nvim-tree has a really nice feature where you can highlight a file (say an excel file) and press `s` which will then open the file using the computer's default program for excel files. (Such as Microsoft Excel or LibreOffice.)

<img width="308" alt="Screenshot 2023-08-11 at 8 10 38 PM" src="https://github.com/nvim-neo-tree/neo-tree.nvim/assets/22198719/a5b444ad-b0f9-45d0-9042-b046353f06a7">

This feature is amazing. Does neo-tree.nvim have this same functionality? I couldn't find it in the docs, nor when I was using it.

**Comment by jjiangweilan** [comment]
any update

**Reply by HiramTheHero** [reply]
There is an example in the Wiki for Linux and Mac: [https://github.com/nvim-neo-tree/neo-tree.nvim/wiki/Recipies#open-with-system-viewer](https://github.com/nvim-neo-tree/neo-tree.nvim/wiki/Recipies#open-with-system-viewer)

(https://github.com/nvim-neo-tree/neo-tree.nvim/issues/1112#issuecomment-1686414230)
